integration-libs/opf/global-functions/core/facade/opf-global-functions.service.ts

```diff
@@ -333,4 +340,14 @@ export class OpfGlobalFunctionsService implements OpfGlobalFunctionsFacade {
       );
     };
   }
+  protected registerCtaScriptReady(domain: GlobalFunctionsDomain): void {
+    this.getGlobalFunctionContainer(domain).scriptReady = (
+      scriptIdentifier: string
+    ): void => {
+      console.log('flo', scriptIdentifier);
+      this.ngZone.run(() => {
+        this.opfCtaFacade.emitScriptReadyEvent(scriptIdentifier);
```

Comment:
We use a Subject to communicate between 3rd party script (which emit) and Spartacus (which subscribe).
No prob while testing.